augraphy/augmentations/watermark.py

```diff
@@ -128,22 +128,99 @@ def apply_watermark(self, watermark_foreground, image):
             watermark_location = self.watermark_location
 
         # initialize watermark method
-        if self.watermark_method == "random":
-            watermark_method = ["darken", "mix", "normal"]
+        if self.watermark_method == "random" or self.watermark_method not in ["overlay", "obfuscate"]:
+            watermark_method = random.choice(["overlay", "obfuscate"])
         else:
             watermark_method = self.watermark_method
 
-        # overlay watermark foreground and input image
-        ob = OverlayBuilder(
-            watermark_method,
-            watermark_foreground,
-            image,
-            ntimes=1,
-            nscales=(1, 1),
-            edge=watermark_location,
-            edge_offset=10,
-            alpha=0.3,
-        )
+        if watermark_method == "obfuscate":
```

Comment:
In this new watermark_method, certain line of text in the input image will obfuscate the watermark effect, some examples:

![image](https://user-images.githubusercontent.com/52521979/164981476-0ba25549-b076-4067-8f99-17d48c4aa2d7.png)

![image](https://user-images.githubusercontent.com/52521979/164981487-feb458ca-1ccd-4d4d-b620-cdc530fce807.png)

![image](https://user-images.githubusercontent.com/52521979/164981628-199b976e-23cc-46cb-91e6-b47697c24796.png)